### ISSUE TITLE
Failed to load installed themes

### ISSUE THREAD

**Issue by anselal** [VundleVim/Vundle.vim#868]
I have installed the 2 themes via Vundle but vim cannot load them. My `.vimrc` looks like this:

 .vimrc
```
...
Plugin 'jnurmine/Zenburn'
Plugin 'altercation/vim-colors-solarized'

colorscheme zenburn
...
```

I install the plugins with `:PluginInstall` inside vim or with `vim +PluginInstall +qall` from the command line. 

When I open vim I get the following error message

```
E185: Cannot find color scheme 'zenburn'
```

Vundle works fine with other plugings.

To get the themes to work I will have to manually copy over the theme files to `~/.vim/colors` 

eg. `cp ~/.vim/bundle/Zenburn/colors/zenburn.vim ~/.vim/colors`

IS this the default behaviour of Vundle ?

**Comment by davidlowryduda** [comment]
I have no problem loading the zenburn or solarized colorschemes after installing using vundle.

Is that vimrc supposed to represent your entire vimrc? If so, then that is your problem. You should read the "Configure Plugins" section of the [README](https://github.com/VundleVim/Vundle.vim) again and add a complete vundle setup to your vimrc.

**Comment by anselal** [comment]
No this is just a part of my configuration. I have copy pasted the config from the readme and as I said other plugins are working as expected.

**Comment by anselal** [comment]
I updated my config file so that the required code is on top of the file. I had some other configurations on top like `syntax on` & `set number` which do not require any plugins.

I removed my themes from ~/.vim/colors folder and still the themes wouldn't load:

> E185: Cannot find color scheme 'zenburn'

Then I deleted the installed theme from `~/.vim/bundle` and installed it again with `:PluginInstall` and it worked.

I don't know what caused this issue but it is resolved now so I am closing this issue.

Thank you @davidlowryduda for the help !!!
